src/sbt-test/dependencies/compile/etc/bom.xml

```diff
@@ -6,9 +6,15 @@
         <name>CycloneDX SBT plugin</name>
       </tool>
     </tools>
+    <component type="library" bom-ref="pkg:maven/dependencies/dependencies_2.12@0.1">
+      <group>dependencies</group>
+      <name>dependencies_2.12</name>
+      <version>0.1</version>
+      <purl>pkg:maven/dependencies/dependencies_2.12@0.1</purl>
+    </component>
   </metadata>
   <components>
-    <component type="library" bom-ref="pkg:maven/org.scala-lang/scala-library@2.12.20">
+    <component type="library" bom-ref="pkg:maven/org.scala-lang/scala-library@2.12.20?type=jar">
```

Comment:
perhaps we could omit the type when it equals `jar`, since it's the default packaging?